src/components/MaplibreMap.jsx

```diff
@@ -201,15 +201,13 @@ const MaplibreMap = ({ view, setView }) => {
     const map = mapRef.current.getMap()
     const { lat, lng } = map.getCenter()
     const zoom = map.getZoom()
-    const queryParams = getURLParams()
```

Comment:
Here was reusing the `queryParams` name. Not a big deal, but can become confusing for the human.